setup.py

```diff
@@ -54,7 +56,8 @@
     data_files=[(
         (BASE_DIR, ['data/nssm_original.exe'])
     )],
-    install_requires=['indy-plenum==1.13.0.dev169',
+    install_requires=['indy-plenum==1.13.0.dev1034',
```

Comment:
Change to `indy-plenum==1.13.0.dev169`

Reply:
no problem :D

Reply:
Strangely the test collection will fail with the newer version of plenum.

```
_____________________________________________________________________ ERROR collecting indy_node/test/auth_rule/auth_framework/test_auth_rule_using.py _____________________________________________________________________
ImportError while importing test module '/indy-node/indy_node/test/auth_rule/auth_framework/test_auth_rule_using.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
indy_node/test/auth_rule/auth_framework/test_auth_rule_using.py:7: in <module>
    from indy_node.test.auth_rule.auth_framework.edit_frozen_ledgers import EditFrozenLedgersTest
indy_node/test/auth_rule/auth_framework/edit_frozen_ledgers.py:10: in <module>
    from plenum.test.freeze_ledgers.helper import sdk_send_freeze_ledgers
/usr/local/lib/python3.5/dist-packages/plenum/test/freeze_ledgers/helper.py:3: in <module>
    from indy.ledger import build_ledgers_freeze_request, build_get_frozen_ledgers_request
E   ImportError: cannot import name 'build_ledgers_freeze_request'
________________________________________________________________________ ERROR collecting indy_node/test/freeze_ledgers/test_send_ledgers_freeze.py ________________________________________________________________________
ImportError while importing test module '/indy-node/indy_node/test/freeze_ledgers/test_send_ledgers_freeze.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
indy_node/test/freeze_ledgers/test_send_ledgers_freeze.py:4: in <module>
    from plenum.test.freeze_ledgers.helper import sdk_get_frozen_ledgers, sdk_send_freeze_ledgers
/usr/local/lib/python3.5/dist-packages/plenum/test/freeze_ledgers/helper.py:3: in <module>
    from indy.ledger import build_ledgers_freeze_request, build_get_frozen_ledgers_request
E   ImportError: cannot import name 'build_ledgers_freeze_request'
```
In the [run](https://github.com/hyperledger/indy-node/actions/runs/1504681011) without the new plenum versions the test were running.

Reply:
This is a sign that the version of the `indy-sdk` being installed does not contain `indy.ledger.build_ledgers_freeze_request`, but the version of node contains the `freeze_ledgers` tests that require it.

Reply:
This should be resolved when this PR is merged; https://github.com/hyperledger/indy-plenum/pull/1578